docs/docs/components.mdx

```diff
@@ -147,7 +147,7 @@ modeling hierarchical intent structure, use the following flags with any tokeniz
 
 * **Short**
 
-  Tokenizer using whitespaces as a separator
+  Tokenizer using white spaces as a separator
```

Comment:
This is actually correct as a full word, [whitespace ](https://en.wikipedia.org/wiki/Whitespace_character) is a known character in programming